commands/work/work.go

```diff
@@ -0,0 +1,120 @@
+package work
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/taskcluster/slugid-go/slugid"
+	cmd_ep "github.com/taskcluster/taskcluster-worker/commands/extpoints"
+	"github.com/taskcluster/taskcluster-worker/config"
+	"github.com/taskcluster/taskcluster-worker/engines/extpoints"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+	"github.com/taskcluster/taskcluster-worker/worker"
+)
+
+func init() {
+	cmd_ep.CommandProviders.Register(cmd{}, "work")
+}
+
+type cmd struct{}
+
+func (cmd) Summary() string {
+	return "Start the worker."
+}
+
+func (cmd) Usage() string {
+	return `Usage:
+  taskcluster-worker work <engine> --logging-level <level>
```

Comment:
Hmm... My thinking was that it's a required argument... and not an option.
But I see no basis for that in:
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html
or
http://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html